src/components/Garden/ModalFlows/SupportProposal/SupportProposalScreens.js

```diff
@@ -1,16 +1,39 @@
 import React, { useMemo, useState, useCallback } from 'react'
+import { useHistory } from 'react-router'
+import { Button } from '@1hive/1hive-ui'
 import ModalFlowBase from '../ModalFlowBase'
 import ChangeSupport from './ChangeSupport'
 import SupportProposal from './SupportProposal'
 
+// import { buildGardenPath } from '@utils/routing-utils'
 import useActions from '@hooks/useActions'
 
 function SupportProposalScreens({ proposal, mode }) {
   const [transactions, setTransactions] = useState([])
+  const history = useHistory()
   const { convictionActions } = useActions()
 
   const { id: proposalId } = proposal
 
+  const handleViewProposal = useCallback(() => {
+    const path = '/profile'
+    history.push(path)
+  }, [history])
+
+  const renderOnCompleteActions = useCallback(() => {
+    if (mode === 'support' ? 'Support this proposal' : 'Change support') {
+      return (
+        <Button
+          label="View supported proposals"
+          mode="strong"
+          onClick={handleViewProposal}
+          wide
+        />
+      )
+    }
+    return null
+  }, [handleViewProposal, mode])
+
```

Comment:
I am not convinced by this behavior. What do you think of instead change to the main view of the specific Garden?

Reply:
What you think of skip this change from the current PR and instead create a separate one to address it?

Reply:
Agree, this is not ideal.

So I think the natural tendency is to look for confirmation that the support has indeed been placed, with the amount specified.

The home page doesn't quite achieve that.

My initial solution was to simply refresh the page, but unfortunately that disconnects the account.

Ideally, one would just make the modal disappear.

Reply:
Yes, maybe we can make the modal disappear and integrate something like [blocknative's notify](https://docs.blocknative.com/notify) showing a toast at the bottom of the page.

Reply:
Ok, removing so we can merge. will address in another PR